presto-hive/src/main/java/com/facebook/presto/hive/orc/OrcPageSourceFactory.java

```diff
@@ -149,7 +150,7 @@ public static OrcPageSource createOrcPageSource(
             FileSystem fileSystem = hdfsEnvironment.getFileSystem(sessionUser, path, configuration);
             long size = fileSystem.getFileStatus(path).getLen();
             FSDataInputStream inputStream = fileSystem.open(path);
-            orcDataSource = new HdfsOrcDataSource(path.toString(), size, maxMergeDistance, maxBufferSize, streamBufferSize, inputStream, stats);
+            orcDataSource = new HdfsOrcDataSource(new OrcDataSourceId(path.toString()), size, maxMergeDistance, maxBufferSize, streamBufferSize, inputStream, stats);
```

Comment:
I assume the idea is to human-identify source file in case of error. Will the `OrcDataSourceId` produced here differ visually from the one produced in `FileOrcDataSource`? (Or maybe it doesn't matter?)